foundry/tensorflow_dataset_wrapper.py

```diff
@@ -0,0 +1,25 @@
+import numpy as np
```

Comment:
Instead of having two Python files with similar, lengthy names in the same module, let's create a new module. Perhaps something like `foundry.loaders` and have a `tf` and `pytorch` submodule in each. The file layout would look like

```
foundry\
  __init__.py
[ ... ]
 loaders\
    __init__.py
    tf.py
    [ ... ]
```